proto/money/rpc/group_booking_folio_svc.proto

```diff
@@ -102,6 +103,21 @@ message GroupBookingFolioSvcPostCashPaymentRequest {
 	.holms.types.booking.indicators.GroupBookingIndicator booking = 2;
 }
 
+message GroupBookingFolioSvcPostLodgingChargeCorrectionRequest {
+	.holms.types.booking.indicators.GroupBookingIndicator booking = 1;
+	string description = 2;
+	.holms.types.primitive.MonetaryAmount adjustment_subtotal = 3;
+	bool adjustment_taken_from_guest = 4;
+}
+
+message GroupBookingFolioSvcPostInventoryChargeCorrectionRequest {
+	.holms.types.booking.indicators.GroupBookingIndicator booking = 1;
+	.holms.types.supply.incidental_items.IncidentalItemIndicator inventory_item = 2;
+	string description = 3;
+	.holms.types.primitive.MonetaryAmount adjustment_subtotal = 4;
+	bool adjustment_taken_from_guest = 5;
```

Comment:
I'd name this as "adjustment_charged_against_group".

This field is basically a glorified negative sign pointing which way the money is flowing.